docs/contribution/accessibility-testing.md

```diff
@@ -0,0 +1,73 @@
+# Accessibility Testing Guidelines
+
+Spirit Web React components now include automated accessibility checks powered by [axe-core](https://github.com/dequelabs/axe-core) and [`jest-axe`](https://github.com/nickcolley/jest-axe). This document explains how to extend the existing coverage and how to handle known false positives.
+
+## Test Harness
+
+- Jest configuration automatically registers `jest-axe` matchers (including `toHaveNoViolations`) and a custom `toHaveNoAxeViolations` matcher with richer failure output.
+- Reusable helpers live in `packages/web-react/tests/testUtils`. Import `runAxe` from `@local/tests` in component tests.
+- Unit tests can be executed with:
+
+  ```sh
+  yarn workspace @lmc-eu/spirit-web-react test:unit
+  ```
+
+  To focus on accessibility specs only, pass `--testPathPatterns=accessibility`.
+
+## Authoring Accessibility Specs
+
+Follow the render → analyze → assert pattern:
+
+```tsx
+import { render, screen } from '@testing-library/react';
+import { runAxe } from '@local/tests';
+import Button from '../Button';
+
+it('is accessible in its default state', async () => {
```

Comment:
suggestion(if-minor): Please, use the description in the form of "it should do something", e.g. `it('should ...')` - in this case, "it should be accessible in its default state".